packages/libs/react-ui/src/components/IconButton/IconButton.css.ts

```diff
@@ -0,0 +1,68 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import {
+  createVar,
+  fallbackVar,
+  globalStyle,
+  style,
+  styleVariants,
+} from '@vanilla-extract/css';
+
+const surfaceColor = createVar(),
+  highContrastColor = createVar();
+
+export const container = style([
+  sprinkles({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 'lg',
+    cursor: 'pointer',
+    size: 11,
+  }),
+  {
+    backgroundColor: fallbackVar(surfaceColor, 'transparent'),
+    color: fallbackVar(highContrastColor, vars.colors.neutral5),
+    border: 0,
```

Comment:
I believe this can be a sprinkle

Reply:
Sprinkles doesn't seem to like the variables created via `createVar` for backgroundColor nor color.

Border was good to move into a sprinkle